status-im/status-react
# Send abridged version of group chats

Fixes: #11602 
When sending a message in a private group chat we send the whole history
    for redundancy and allow out-of-order processing.
    This can be very expensive in some chats, resulting in long delay when
    sending a message and calculating the POW.
    This commit improves the performance  by only forwarding the events
    necessary for the user to be able to construct a group chat and process
    correctly the message.

### Testing

Existing group chats, creating new group chats.


### Scenario to test

1) Create account A1
2) Create account B1
3) Create account C1
3) Exchange some 1-to-1 messages from A1 to B1, A1 to C1 and B1 to C1
4) Create private group on A1 (admin) including B1 & C1
4) Create account A2
5) Send message in private group chat from A1 

What do you see on A2?
